common/src/main/java/org/apache/rocketmq/common/protocol/RequestCode.java

```diff
@@ -17,7 +17,9 @@
 
 package org.apache.rocketmq.common.protocol;
 
-public class RequestCode {
+import org.apache.rocketmq.remoting.protocol.RemotingSysRequestCode;
+
```

Comment:
ResponseCode in common module has already had remoting as its dependency as it extends RemotingSysResponseCode, which is a class of remoting module.